website/project/model.py

```diff
@@ -1285,7 +1290,7 @@ def update(self, fields, auth=None, save=True):
                     value.get('year'),
                     value.get('copyright_holders'),
                     auth,
-                    save=False
+                    save=True
```

Comment:
It probably makes sense to pass `save=save` here, so we don't have any unwanted implicit saves. @barbour-em Can you make that change and make sure this still works as expected?